.dockerignore

```diff
@@ -0,0 +1,3 @@
+postgres-data/
```

Comment:
Let's create an `.gitignore` file and add the `postgres-data/` dir to that gitignore file